metadata.json

```diff
@@ -21,9 +21,6 @@
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "5",
-        "6",
-        "7",
```

Comment:
@bastelfreak I didn't think EL7 went EOL until 30th of June?

Reply:
I still need a bit of time for the next release and I think it doesn't make sense to make a second major release a month later just for EL7.